arangod/RestHandler/RestAnalyzerHandler.cpp

```diff
@@ -220,31 +219,22 @@ void RestAnalyzerHandler::createAnalyzer( // create
 }
 
 arangodb::RestStatus RestAnalyzerHandler::execute() {
-  auto* analyzers = arangodb::application_features::ApplicationServer::lookupFeature< // find feature
-    IResearchAnalyzerFeature // feature type
-  >();
-
-  if (!analyzers) {
-    generateError( // generate error
-      arangodb::rest::ResponseCode::BAD, // HTTP code
-      TRI_ERROR_INTERNAL, // code
-      std::string("failure to find feature '") + IResearchAnalyzerFeature::name() + "' while executing REST request for: " + ANALYZER_PATH
-    );
-
-    return arangodb::RestStatus::DONE;
-  }
-
-  auto* sysDatabase = arangodb::application_features::ApplicationServer::lookupFeature< // find feature
-    arangodb::SystemDatabaseFeature // featue type
-  >();
-  auto sysVocbase = sysDatabase ? sysDatabase->use() : nullptr;
-
   if (!_request) {
     generateError( // generate error
       arangodb::rest::ResponseCode::METHOD_NOT_ALLOWED, // HTTP code
       TRI_ERROR_HTTP_BAD_PARAMETER // error code
     );
+    return arangodb::RestStatus::DONE;
+  }
 
+  auto* analyzers = 
+    arangodb::application_features::ApplicationServer::lookupFeature<IResearchAnalyzerFeature>();
```

Comment:
I suggest using `getFeature<...>(...)` here.
If the feature doesn't exist, this will throw an exception, which, if not caught, will be turned into a nice REST error response automatically. So you can safe several lines of code here.